src/main/java/org/folio/domain/entity/EcsTlrEntity.java

```diff
@@ -10,21 +14,29 @@
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 import lombok.ToString;
-import org.folio.domain.converter.UUIDConverter;
-import java.util.UUID;
 
 @Entity
-@Table(name = "title_level_request")
+@Table(name = "ecs_tlr")
 @Getter
 @Setter
 @ToString
 @AllArgsConstructor
 @NoArgsConstructor
 @Builder
-public class TitleLevelRequestEntity {
+public class EcsTlrEntity {
```

Comment:
`@Getter @Setter @ToString` can be replaced with `@Data`

Reply:
updated